### ISSUE TITLE
dev_settings.py

### ISSUE THREAD

**Issue by flexgrip** [sgricci/dewdrop#24]
Hello,

After making a few adjustments, I am able to run app.py and get a login window. But I'm assuming I need to copy dev_settings.sample.py to dev_settings.py and then fill in all the API key stuff... right?

**Comment by nehpe** [comment]
Yep yep, you got it. They are used here: https://github.com/sgricci/dewdrop/blob/master/libdroplr.py
